### DISCUSSION TITLE
Include time since download/updated

### DISCUSSION THREAD

**Discussion by rjonkman** [bpozdena/OneDriveGUI#192] [Ideas]
I often save several changes to a file over the course of an hour on macOS or Windows, and then I'll switch to Linux but I won't know if the file has updated to the latest revision because the OneDriveGUI only shows that the the file is 'available'.  Please include how much time has elapsed since the file was last updated.  Keep up the good work.

**Comment by jgelcon** [comment]
With version 1.1.1 it already shows "Downlaoded" or "Uploaded" instead of "available", which in some cases is enough to know if the file has been updated. Thanks a lot for that!
But I do agree that in the future adding the date and hour would be a nice improvement.

**Comment by bpozdena** [comment]
Will be implemented in v1.3.0
